lib/apartment/log_subscriber.rb

```diff
@@ -1,56 +1,19 @@
 # frozen_string_literal: true
 
 module Apartment
-  class LogSubscriber < ActiveSupport::LogSubscriber
-    IGNORE_PAYLOAD_NAMES = ["SCHEMA", "EXPLAIN"]
-
-    class_attribute :backtrace_cleaner, default: ActiveSupport::BacktraceCleaner.new
-
-    def self.runtime=(value)
-      ActiveRecord::RuntimeRegistry.sql_runtime = value
-    end
-
-    def self.runtime
-      ActiveRecord::RuntimeRegistry.sql_runtime ||= 0
-    end
-
-    def self.reset_runtime
-      rt, self.runtime = runtime, 0
-      rt
-    end
-
+  class LogSubscriber < ActiveRecord::LogSubscriber
     def sql(event)
```

Comment:
Do we need to define the `sql` method?

Reply:
I also thought we would not need it, but removing it makes it not work anymore